src/views/PurchaseOrders.vue

```diff
@@ -33,8 +32,33 @@ export default defineComponent({
     IonToolbar,
     PurchaseOrderItem
   },
+  computed: {
+    ...mapGetters({
+      orders: 'purchaseOrder/getPurchaseOrders',
+      totalOrders: 'purchaseOrder/getTotalPurchaseOrder'
+    })
+  },
+  methods: {
+    async getPurchaseOrder(){
+      const paylaod = {
+        "json": {
+          "params": {
+            "rows": 10,
```

Comment:
Why we have hardcoded rows value, suppose if we have 50 PO then we will always get the first 10, need to make it dynamic.

Reply:
Made it dynamic Sir.